dojo/tools/acunetix/parser_helper.py

```diff
@@ -175,12 +177,20 @@ def get_defectdojo_findings(filename):
     for report_item in acunetix_scan_report.ReportItems:
         defectdojo_finding = dict()
 
-        defectdojo_finding['title'] = report_item['Name']
+        if "Affects" in report_item:
+            affects = (" ({})".format(report_item['Affects']))
+        else:
+            affects = ""
+        defectdojo_finding['title'] = "{}{}".format(report_item['Name'], affects)
```

Comment:
Changing the title format will break deduplication as the acuentix parser does not yet use the new DEDUPLICATION_ALGORITHM_PER_PARSER config in settings.py.

Reply:
What do you recommend to do in that case?
- Should I not update the title? Doing this will remove similar vulnerabilities on different endpoints
- Modify the deduplication algorithm in settings.py?
- Something else?

Reply:
Best would be to modify/set the dedupe algorithm in settings.dist.py

Reply:
I made an update. Hope it's all good